client/templates/HomeView.ejs

```diff
@@ -127,6 +128,14 @@
             margin-right:32px;
         }
 
+        #content-middle {
+            margin-bottom: 2px;
+        }
+
+        #content-top-right {
+            margin-top: 22px;
```

Comment:
This should be 24px as needs to follow the rules of having spacings to be one of 2px, 4px, 8px, 16px, 24px, 32px, and 64px